### DISCUSSION TITLE
Recreating ES indexes

### DISCUSSION THREAD

**Discussion by dcore94** [Netflix/conductor#3286] [Q&A]
I'm running conductor as a docker stack. Everytime I restart the stack ES looses the indexed data and the executions panel in the UI appears to be empty.
Is there a way to force recreation of the lost indexes in ES?
Thank you.

**Comment by Prebiusta** [comment]
Have you considered saving elastic data on the volume? That would solve the problem with loosing data after restarting

**Reply by dcore94** [reply]
I thought data is kept in memory by ES .... If it is written to disk then this is doable indeed... 
Not very easy because running on a docker-swarm I need to ask our sysadmin to create an NFS volume among all cluster nodes but we'll try it out.

**Reply by Prebiusta** [reply]
You are right, ES keeps data in memory. And that's why when you restart, it stops and starts the container, so memory is wiped out. That's why it is important to persist data elsewhere, e.g. volumes.
